src/settings/_typography-font-families.scss

```diff
@@ -1,12 +1,38 @@
+@import "compatibility";
+
 ////
 /// @group settings/typography
 ////
 
-/// List of font families to use if using NTA (the default font 'stack' for
+/// Use 'legacy' fonts
+///
+/// Whether or not to use v1 nta font from GOV.UK Elements / Frontend
+/// Toolkit, for teams that are migrating to GOV.UK Frontend and may be using
+/// components from both places in a single application.
+///
+/// @type Boolean
+/// @access public
+
+$govuk-use-legacy-font: if((
```

Comment:
This doesn't get used in this file – would it make more sense in `typography-font`?